src/Faker/Provider/ar_MA/Company.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Faker\Provider\ar_MA;
+
+class Company extends \Faker\Provider\Company
+{
+    protected static $formats = array(
+        '{{lastName}} {{companySuffix}}',
+        '{{companyPrefix}} {{lastName}} {{companySuffix}}',
+        '{{companyPrefix}} {{lastName}}',
+    );
+
+    protected static $bsWords = array(
+        array()
+    );
+
+    protected static $catchPhraseWords = array(
+        array('الخدمات','الحلول','الانظمة'),
+        array(
+            'الذهبية','الذكية','المتطورة','المتقدمة', 'الدولية', 'المتخصصه', 'السريعة',
+            'المثلى', 'الابداعية', 'المتكاملة', 'المتغيرة', 'المثالية'
+            ),
```

Comment:
Seems like the indentation is a bit off here:
```suggestion
        ),
```